packages/react-button/package.json

```diff
@@ -42,7 +42,9 @@
   },
   "dependencies": {
     "@fluentui/keyboard-key": "^0.3.1",
+    "@fluentui/react-icons-mdl2": "^1.1.1",
     "@fluentui/react-make-styles": "^9.0.0-alpha.38",
+    "@fluentui/react-shared-contexts": "^9.0.0-alpha.12",
```

Comment:
This dep shouldn't be needed anymore, you don't seem to use the it anywhere

Reply:
Removed!